src/lib/Sendbird.tsx

```diff
@@ -116,7 +119,9 @@ function Sendbird(props: SendbirdProviderProps) {
              * we convert it from here.
              * i.e. 'THREAD' -> 'thread'
              */
-            replyType: replyType.toLowerCase(),
+            replyType: replyType == null
+              ? initialConfig.groupChannel.channel.replyType
+              : replyType.toLowerCase() as GroupChannelConfig['channel']['replyType'],
```

Comment:
Probably better to be moved out to a function, as discussed in older PR , but is not a dealbreaker

Reply:
Oh probably I missed the comment that you commented earlier. Do u remember what you suggested?

Reply:
No, I thiknk its @bang9  suggested it

Reply:
I see. Let me merge this one first (your dev env is broken, right?) and look the comment up.